README.md

```
@@ -1,37 +1,145 @@
-# Deploy Agent Client
+# Deploy Agent
 
-This gem allows you to configure a secure proxy through which DeployHQ can forward connections.
+A secure proxy that allows [DeployHQ](https://www.deployhq.com/) to forward connections to servers behind firewalls. The agent establishes an outbound TLS connection to DeployHQ's servers and proxies deployment traffic to allowed destinations based on an IP/network allowlist.
 
-## Installation
+## How It Works
 
-You'll need Ruby installed on your system. We've tested on 2.7.8 and later.
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)